packages/roosterjs-editor-plugins/lib/plugins/TableResize/editors/TableEditor.ts

```diff
@@ -140,7 +140,8 @@ export default class TableEditor {
                             true /*isHorizontal*/
                         );
                     } else {
-                        this.setInserterTd(null);
+                        this.setInserterTd(null, true);
```

Comment:
Thanks for the fix. 

Can we fix inside setInsertTd() instead, when td is null, we always hide both inerters?

Reply:
It's ok but better describe this default behavior in comment

Reply:
yes, feel free to add comment